GAE/src/org/waterforpeople/mapping/serialization/response/MediaResponse.java

```diff
@@ -67,6 +68,34 @@ public static String format(String value, int version) {
         return media.getFilename();
     }
 
+
+    /**
+     * Format media value as JSON with a geotag location, whether null or not.
+     */
+    public static String formatWithGeotag(Media media) {
+            try {
+                return jsonObjectWriter.writeAsString(media);
+            } catch (IOException e) {
+                log.warning(e.getMessage());
+                return "";
+            }
+        }
+
+
```

Comment:
Maybe too many spaces? Maybe use the IDE automatic formatting shortcuts?

Reply:
I like two empty lines between methods.